app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/vault/util/VaultAddItemStateExtensions.kt

```diff
@@ -211,3 +214,17 @@ private fun List<UriItem>?.toLoginUriView(): List<LoginUriView>? =
         ?.filter { it.uri?.isNotBlank() == true }
         ?.map { LoginUriView(uri = it.uri.orEmpty(), match = it.match, uriChecksum = null) }
         .takeUnless { it.isNullOrEmpty() }
+
+private fun getRevisionDate(
```

Comment:
Can we make this an extension on `VaultAddEditState.ViewState.Content.ItemType.Login`